CONTRIBUTING.md

```diff
@@ -56,4 +56,36 @@ If you agree to this for your contribution, then all that's needed is to include
 Signed-off-by: Your Name <your@email.example.org>
 ```
 
-Git allows you to add this signoff automatically when using the `-s` flag to `git commit`, which uses the name and email set in your `user.name` and `user.email` git configs.
\ No newline at end of file
+Git allows you to add this signoff automatically when using the `-s` flag to `git commit`, which uses the name and email set in your `user.name` and `user.email` git configs.
```

Comment:
Do we need signoff for contributions to the matrix.org website?

Reply:
Iirc this was added before and is already on Main. So I argue yes 😅